src/options.js

```diff
@@ -0,0 +1,14 @@
+import path from 'path';
+import pkg from './package.json';
+
+export default {
+  // cmd, homepage, bugs all pulled from package.json
+  cmd: 'sagastandard',
+  version: pkg.version,
+  homepage: pkg.homepage,
+  bugs: pkg.bugs.url,
+  eslint: require('eslint'),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a standard "import" statement instead of "require(...)". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS3533)